artifactory/commands/transferfiles/transferfileprogress.go

```diff
@@ -112,6 +127,10 @@ func (t *TransferProgressMng) Quit() error {
 		if t.totalRepositories != nil {
 			t.barsMng.QuitTasksWithHeadlineProg(t.totalRepositories)
 		}
+		if t.totalSize != nil {
+			t.totalSize.GetBar().Abort(true)
+			t.totalSize = nil
+		}
```

Comment:
Any reason why the above code lines aren't placed inside the `abortMetricsBars` function?
The rest of progress bars are aborted there.